Tests/MapboxMapsTests/MapView/Unit Tests/MapInitOptionsTests.swift

```diff
@@ -0,0 +1,142 @@
+import XCTest
+@testable import MapboxMaps
+
+class MapInitOptionsTests: XCTestCase {
+
+    private var dataSourceReturnValue: Any = false
+    private var oldDefaultAccessToken: String = ""
+
+    override func setUp() {
+        super.setUp()
+        oldDefaultAccessToken = CredentialsManager.default.accessToken
+    }
+    override func tearDown() {
+        super.tearDown()
+        CredentialsManager.default.accessToken = oldDefaultAccessToken
+
+        oldDefaultAccessToken = ""
+        dataSourceReturnValue = false
+    }
+
+    func testDefaultMapInitOptionsAreOverridden() {
+        do {
+            let updatedMapInitOptions = MapInitOptions()
+            XCTAssertNotEqual(updatedMapInitOptions.resourceOptions.accessToken, "pk.aaaaaa")
+        }
+
+        CredentialsManager.default.accessToken = "pk.aaaaaa"
+
+        do {
+            let updatedMapInitOptions = MapInitOptions()
+            XCTAssertEqual(updatedMapInitOptions.resourceOptions.accessToken, "pk.aaaaaa")
+        }
+    }
+
+    func testOverridingDefaultCredentialsManagerAccessToken() {
+        CredentialsManager.default.accessToken = "pk.bbbbbb"
+
+        let mapView = MapView(frame: .zero)
+        let resourceOptions = try! mapView.__map.getResourceOptions()
+
+        XCTAssertEqual(resourceOptions, ResourceOptions(accessToken: CredentialsManager.default.accessToken))
+        XCTAssertEqual(resourceOptions.accessToken, CredentialsManager.default.accessToken)
+    }
```

Comment:
This seems like a test of MapView, not MapInitOptions

Reply:
Have moved the integration tests to a new file.